packages/react-instantsearch/src/components/CurrentRefinements.js

```diff
@@ -32,35 +33,36 @@ class CurrentRefinements extends Component {
   }
 
   render() {
-    const { translate, items, refine, canRefine } = this.props;
+    const { translate, items, refine, canRefine, header, footer } = this.props;
 
     return (
-      <div {...cx('root', !canRefine && 'noRefinement')}>
-        <div {...cx('items')}>
-          {items.map(item => (
-            <div key={item.label} {...cx('item', item.items && 'itemParent')}>
-              <span {...cx('itemLabel')}>{item.label}</span>
-              {item.items ? (
-                item.items.map(nestedItem => (
-                  <div key={nestedItem.label} {...cx('item')}>
-                    <span {...cx('itemLabel')}>{nestedItem.label}</span>
-                    <button
-                      {...cx('itemClear')}
-                      onClick={() => refine(nestedItem.value)}
-                    >
-                      {translate('clearFilter', nestedItem)}
-                    </button>
-                  </div>
-                ))
-              ) : (
-                <button {...cx('itemClear')} onClick={() => refine(item.value)}>
```

Comment:
We forgot this part in the refactoring.

Reply:
This part wouldn't be necessary according to the changes I've suggested on the PR desc. Happy to discuss it if needed!

Reply:
Wasn't sure about that, works for me.